tests/support/util.tcl

```diff
@@ -72,13 +72,7 @@ proc sanitizer_errors_from_file {filename} {
 }
 
 proc getInfoProperty {infostr property} {
-    if {[regexp "\r\n$property:(.*?)\r\n" $infostr _ value]} {
-        set _ $value
-    }
-}
-
-proc cluster_info {r field} {
-    if {[regexp "^$field:(.*?)\r\n" [$r cluster info] _ value]} {
+    if {[regexp "$property:(.*?)\r\n" $infostr _ value]} {
```

Comment:
i see you removed the `\r\n` search at the beginning of the line.
this means we can get a sub-string match. e.g. searching for `clients` will find `connected_clients`.
maybe we should add `\$` to start from beginning of a line?
p.s. the old code was ok since the output always starts with a section header.